docs/OVERVIEW.md

```diff
@@ -0,0 +1,339 @@
+img.dask_data  # Chunks use normal ZYX
+```
+
+If the reader does not support stitching tiles together the M tile index will be
+available on the `BioImage` object:
+
+```python
+img = BioImage("some-unsupported-mosaic-stitching-format.ext")
+img.dims.order  # M (tile index), T, C, Z, small Y, small X, (S optional)
+img.dask_data  # Chunks use normal ZYX
+```
+
+#### Reader
+
+If the file format reader detects mosaic tiles in the image, the `BioImage` object
+will store the tiles as a dimension.
+
+If tile stitching is implemented, the `BioImage` can also return the stitched image.
+
+```python
+reader = BioImage("ver-large-mosaic.lif")
+reader.dims.order  # M, T, C, Z, tile size Y, tile size X, (S optional)
+reader.dask_data  # normal operations, can use M dimension to select individual tiles
+reader.mosaic_dask_data  # returns stitched mosaic - T, C, Z, big Y, big, X, (S optional)
+```
+
+#### Single Tile Absolute Positioning
+
+There are functions available on the `BioImage` object
+to help with single tile positioning:
+
+```python
+img = BioImage("very-large-mosaic.lif")
+img.mosaic_tile_dims  # Returns a Dimensions object with just Y and X dim sizes
+img.mosaic_tile_dims.Y  # 512 (for example)
+
+# Get the tile start indices (top left corner of tile)
+y_start_index, x_start_index = img.get_mosaic_tile_position(12)
+```
+
+### Metadata Reading
+
+```python
+from bioio import BioImage
+
+# Get a BioImage object
+img = BioImage("my_file.tiff")  # selects the first scene found
+img.metadata  # returns the metadata object for this file format (XML, JSON, etc.)
+img.channel_names  # returns a list of string channel names found in the metadata
+img.physical_pixel_sizes.Z  # returns the Z dimension pixel size as found in the metadata
+img.physical_pixel_sizes.Y  # returns the Y dimension pixel size as found in the metadata
+img.physical_pixel_sizes.X  # returns the X dimension pixel size as found in the metadata
+```
+
+### Xarray Coordinate Plane Attachment
+
+If `bioio` finds coordinate information for the spatial-temporal dimensions of
+the image in metadata, you can use
+[xarray](http://xarray.pydata.org/en/stable/index.html) for indexing by coordinates.
+
+```python
+from bioio import BioImage
+
+# Get a BioImage object
+img = BioImage("my_file.ome.tiff")
+
+# Get the first ten seconds (not frames)
+first_ten_seconds = img.xarray_data.loc[:10]  # returns an xarray.DataArray
+
+# Get the first ten major units (usually micrometers, not indices) in Z
+first_ten_mm_in_z = img.xarray_data.loc[:, :, :10]
+
+# Get the first ten major units (usually micrometers, not indices) in Y
+first_ten_mm_in_y = img.xarray_data.loc[:, :, :, :10]
+
+# Get the first ten major units (usually micrometers, not indices) in X
+first_ten_mm_in_x = img.xarray_data.loc[:, :, :, :, :10]
+```
+
+See `xarray`
+["Indexing and Selecting Data" Documentation](http://xarray.pydata.org/en/stable/indexing.html)
+for more information.
+
+### Cloud IO Support
```

Comment:
was just looking at the live version and for some reason this got rendered with `#id2` instead of `#cloud-io-support`.  There is a link to it near the top of the document which now seems broken.

Reply:
weird, but adjusted